### PR TITLE
feat.: load about section data

### LINKED ISSUE #41: add about data in departments screen

### PR DESCRIPTION
**Pull request by 2tanayk**
fixes #41

### CODE REVIEW

File: lib/screens/department_screen/department_screen.dart [hunk 1]
```diff
@@ -111,7 +111,10 @@ class _DepartmentScreenState extends State<DepartmentScreen> {
 
   @override
   void dispose() {
-    overlayEntry!.remove();
+    if (overlayEntry != null) {
+      overlayEntry!.remove();
+    }
```
Comment (SirusCodes) [inline lib/screens/department_screen/department_screen.dart:116]: You can use `overlayEntry?.remove();`

File: lib/screens/department_screen/department_screen.dart [hunk 2]
```diff
@@ -111,7 +111,10 @@ class _DepartmentScreenState extends State<DepartmentScreen> {
...
+
     overlayEntry = null;
```
Comment (SirusCodes) [inline lib/screens/department_screen/department_screen.dart:118]: No need to set it to null

File: pubspec.lock [hunk 3]
```diff
@@ -28,7 +28,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.8.2"
+    version: "2.8.1"
```
Comment (SirusCodes) [inline pubspec.lock:31]: Are you on flutter 2.10?

**Review by SirusCodes — CHANGES_REQUESTED** [review]

**Comment by SirusCodes** [comment]
Add which issues it fixes?

File: lib/screens/department_screen/department_screen.dart (hunk 2 above)
Comment (2tanayk) [reply lib/screens/department_screen/department_screen.dart:118]: if it isn't set to null, then say I click the overlay thing multiple times, multiple instances get created and the if condition also gets passed

File: pubspec.lock (hunk 3 above)
Comment (2tanayk) [reply pubspec.lock:31]: yes I have upgraded

**Comment by 2tanayk** [comment]
> Add which issues it fixes?

#41 

> Add which issues it fixes?

#41

**Comment by 2tanayk** [comment]
> Add which issues it fixes?

#41

File: lib/screens/department_screen/department_screen.dart (hunk 2 above)
Comment (SirusCodes) [reply lib/screens/department_screen/department_screen.dart:118]: Cool

**Review by SirusCodes — APPROVED** [review]

**Pull request merged**